modules/@ergo/products/components/Forms/Fields/ProductTemplateFormDateField.vue

```diff
@@ -97,12 +97,12 @@ export default {
             const { isDraft, value } = getMappedValue({
                 data: this.data[attribute_code],
                 draft: this.draft[this.languageCode][attribute_code],
-                defaultValue: '',
+                defaultValue: null,
             });
 
             return {
                 isDraft,
-                value: parseDate(value, DEFAULT_FORMAT, new Date()),
+                value: value ? parseDate(value, DEFAULT_FORMAT, new Date()) : null,
```

Comment:
It's no needed, onle defaultValue as null is enough. -> parseDate(value, DEFAULT_FORMAT, new Date()) is enough